src/DistributedWriteAheadLog/KafkaWALCommon.cpp

```diff
@@ -25,8 +26,24 @@ namespace DWAL
 {
 std::string escapeDWalName(const std::string & namespace_, const std::string & name_)
```

Comment:
we don't need the trailing `_` in the argument name

Reply:
namespace is a keyword in c++ , so I have changed it to name_space